INSTALL

```diff
@@ -15,24 +15,16 @@ limitations under the License.
 
 Detailed build instructions are provided for in both HTML
 and APT format in releases.  The APT sources are 
-available in Subversion checkouts.
+available in Git checkouts.
 
 autotools:
 site/building/autotools.html
 src/site/apt/building/autotools.apt
 
-Apache Ant:
-site/building/ant.html
-src/site/apt/building/ant.apt
+CMake:
```

Comment:
How about using alphabetical order in this file? I've changed it at [the corresponding place](https://github.com/apache/logging-log4cxx/commit/96ef6546993a04e954bee4db4f0791a8138f3e00) as well.